amulet/api/world/world.py

```diff
@@ -228,20 +186,20 @@ def save_iter(
         for dimension, cx, cz in changed_chunks:
             if dimension not in output_dimension_map:
                 continue
-            chunk = self._chunk_history_manager.get_current(
-                dimension, cx, cz, self._block_palette, self._biome_palette
-            )
+            chunk = self._chunk_cache2.get_chunk(dimension, cx, cz)
             if chunk is None:
                 wrapper.delete_chunk(cx, cz, dimension)
             else:
                 wrapper.commit_chunk(chunk, dimension)
+            if chunk.changed:
```

Comment:
You don't need that if, you can just set it

Reply:
You are correct. I couldn't remember the logic behind chunk.changed